soccer/Logger.cpp

```diff
@@ -1,9 +1,12 @@
 #include "Logger.hpp"
```

Comment:
Logger changes are going to cause major conflict with an upcoming PR. Just a heads up, I'll probably just redo these fixes on top of the new version.